Automattic/jetpack
# My Jetpack: fix products card section story

This PR fixes story of the products card section.

Fixes #

#### Changes proposed in this Pull Request:

* My Jetpack: fix products card section story

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Test with Storybook
* Confirm Products Card Section woks (almost) as expected Probably, you need to switch between other stories to get the mocked data properly.

<img width="1013" alt="image" src="https://user-images.githubusercontent.com/77539/156225918-f6a78a2b-5e58-4812-b58e-e16d8ea01ef1.png">
